templates/terraform/custom_expand/computed_subscription_topic.erb

```diff
@@ -27,7 +27,9 @@ func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d T
         } else {
 		// If no full topic given, we expand it to a full topic on the same project
 		fullTopic := fmt.Sprintf("projects/%s/topics/%s", project, topic)
-		d.Set("topic", fullTopic)
+		if err := d.Set("topic", fullTopic); err != nil {
+			return nil, fmt.Errorf("Error reading topic: %s", err)
```

Comment:
Lots of these d.Set errors are "reading". Is that intentional & I'm missing something, or should it be writing/setting? (I've got no idea how we'd change them en-masse, but wouldn't want to make you to have to make that change manually)

Reply:
I spent the whole day Friday searching and replacing these, and about part way through I realized the error message that I copied said `reading` rather than `writing`, so I changed it for some of them. I can change these if you need me to, or at least as many as I can without spending too much time on it.